src/VisualStudio/IntegrationTest/IntegrationTests/AbstractChangeSignatureDialogTest.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Windows.Automation;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Microsoft.VisualStudio.IntegrationTest.Utilities.Input;
+using Roslyn.Test.Utilities;
+
+namespace Roslyn.VisualStudio.IntegrationTests
+{
+    public abstract class AbstractChangeSignatureDialogTest : AbstractEditorTest
+    {
```

Comment:
My preference is for composition rather than inheritance. That is, I'd like to see everything in this class pulled out into a `ChangeSignatureDialog_OutOfProc` type, and then you don't need it at all. That will encourage code reuse, maintainability, and simplicity.